src/test/mockData.ts

```diff
@@ -98,8 +98,8 @@ export const mockData: MockCity =
             food: {
                 color: "green",
                     ideas: [{
-                            name: "idea name",
-                            description: "lorem ipsum dolor sit amet ...",
+                            name: "Guachinche Ivan",
+                            description: "Go to Guachinche Ivan in La Laguna and take a rabbit",
                             coordinates:{
                               "type": "Feature",
                               "properties": {},
```

Comment:
Same here, no coma. 🏺

Reply:
sorry this not deserves a change requested